lib/zlib.js

```diff
@@ -54,8 +53,8 @@ const codes = {
 };
 
 const ckeys = Object.keys(codes);
-for (var ck = 0; ck < ckeys.length; ck++) {
-  var ckey = ckeys[ck];
+
+for (const ckey of ckeys) {
```

Comment:
See https://github.com/davidmarkclements/v8-perf#iterating-over-objects for some recent benchmarks on object iteration. Thought it doesn't include `for-of`, I wouldn't expect it to beat `Object.keys` + `for`.